proposals/0000-exception-backtraces.rst

```diff
@@ -0,0 +1,553 @@
+Decorate exceptions with backtrace information
+==============================================
+
+.. authors:: Ben Gamari; David Eichmann; Sven Tennie
+.. date-accepted::
+.. ticket-url:: https://gitlab.haskell.org/ghc/ghc/issues/18159
+.. implemented:: in-progress <https://gitlab.haskell.org/ghc/ghc/-/merge_requests/8869>
+.. highlight:: haskell
+.. header:: This proposal is `discussed at this pull request <https://github.com/ghc-proposals/ghc-proposals/pull/330>`_.
+.. contents::
```

Comment:
This proposal clearly predates the addition of section numbers to the proposal template. :grin: 

```suggestion
.. sectnum::
.. contents::
```